erigon-lib/diagnostics/stages.go

```diff
@@ -82,6 +82,14 @@ func (ti CurrentSyncSubStage) Type() Type {
 	return TypeOf(ti)
 }
 
+type StopNodeEvent struct {
+	Stop bool
+}
+
+func (ti StopNodeEvent) Type() Type {
```

Comment:
Why to use type for event while before type is used for stages?

Reply:
I forgot to delete this, it isn't in use at all.